code/game/gamemodes/events/powercreeper.dm

```diff
@@ -137,7 +137,11 @@
 		return 0
 	Beam(M, "lighting", 'icons/obj/zap.dmi', 5, 2)
 	playsound(src,'sound/weapons/electriczap.ogg',50, 1) //we still want a sound
-	return electrocute_mob(M, powernet, src)
+	if(!electrocute_mob(M, powernet, src))
+		var/damage = (10/(-1*(powernet.avail / 5000)))+45
+		if(damage > 0) //dont wanna heal em, damage only starts at around ~1kW
+			M.apply_damage(M.apply_damage(damage, BURN));
```

Comment:
If the damage value is negative it will heal the person instead, damage > 0 makes the powercreeper heal people

Reply:
my man you need healing damage > 0 only lets through positive values

Reply:
I had a brain fart, sorry